### ISSUE TITLE
Object.keys should be included in prefer-lodash-method

### ISSUE THREAD

**Issue by gajus** [wix/eslint-plugin-lodash3#8]
If I am not mistaken, `Object.keys({})` should throw an error when `prefer-lodash-method` rule is enabled.

**Comment by gajus** [comment]
If not, it would nice to have such a rule.

**Comment by ganimomer** [comment]
That is a valid point.
Just released v0.3.0 with a fix that also reports problems when using static methods that Lodash implements.

**Comment by gajus** [comment]
:+1:

Just to put it on the record for the others finding this issue, the supported methods:

```
Object: ['assign', 'create', 'keys', 'values'],
Array: ['isArray']
['every', 'fill', 'filter', 'find', 'findIndex', 'forEach', 'includes', 'indexOf', 'lastIndexOf', 'map', 'reduce', 'reduceRight', 'some']
```

Extracted from this commit, https://github.com/wix/eslint-plugin-lodash3/commit/7bcd3f2720ea726585fb4acd6f42367a43f47483.
